src/algorithms/conditional_algorithm.h

```diff
@@ -2,19 +2,34 @@
 #define BAYESMIX_ALGORITHMS_CONDITIONAL_ALGORITHM_H_
 
 #include <Eigen/Dense>
+#include <memory>
 
 #include "base_algorithm.h"
 #include "src/collectors/base_collector.h"
+#include "src/mixings/conditional_mixing.h"
 
 class ConditionalAlgorithm : public BaseAlgorithm {
+ protected:
+  //! Weights of clusters
+  Eigen::VectorXd weights;
```

Comment:
Why do we need this member in the Algorithm class? I think we should always refer to the Mixing to get the weights

Reply:
Right, it's just leftovers from old scraped ideas. This class is still basically unimplemented. I will remove this.